kuksa_databroker/databroker/tests/read_write_values.rs

```diff
@@ -75,9 +74,10 @@ fn get_data_entries_from_table(
     data_entries
 }
 
-#[given(regex = "^a running Databroker server.*$")]
-async fn start_databroker_server(w: &mut DataBrokerWorld, step: &Step) {
-    w.start_databroker(get_data_entries_from_table(step)).await;
+#[given(regex = "^a running Databroker server with authorization (true|false).*$")]
```

Comment:
can we use `(enabled|disabled)` instead? That would make the scenario descriptions more *readable* FMPOV ...